src/Sylius/Bundle/GridBundle/DependencyInjection/Configuration.php

```diff
@@ -38,6 +39,20 @@ public function getConfigTreeBuilder()
     /**
      * @param ArrayNodeDefinition $node
      */
+    private function addDriversSection(ArrayNodeDefinition $node)
+    {
+        $node
+            ->children()
+                ->arrayNode('drivers')
```

Comment:
Not sure how ambiguous this is -- I assume the user could equally register their own drivers, in whch case they would not need to include them here.

Should add an `->info` element.